### PR TITLE
add docker info to README.md

### PR DESCRIPTION
**Pull request by whatwareweb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,25 @@ However, on-device debugging is hard, so the engine can also be compiled to run
 
 ## Building
 
+### Using Docker (Easiest for beginners)
```
Comment (JosiasAurel) [inline README.md]: can we remove the `(Easiest for beginners)`? It helps even pros :)

File: README.md [hunk 2]
```diff
@@ -8,6 +8,25 @@ However, on-device debugging is hard, so the engine can also be compiled to run
...
+
+#### Prerequisites:
+ - Working Linux/MacOS system with Docker environment
+ - Dependencies commands:
+   - docker
+   - echo
+   - whoami
+   - grep
```
Comment (JosiasAurel) [inline README.md]: * echo
* whoami
* grep

already ship with unix systems so i find it unecessary to mention these. They're also not dependencies for Docker

File: README.md [hunk 3]
```diff
@@ -8,6 +8,25 @@ However, on-device debugging is hard, so the engine can also be compiled to run
...
+   - chcon (Linux only)
+
+#### Building
+ - Clone the repository
+ - Change code as desired
+ - Change into the spade directory
+ - Run `build-with-docker.sh`
```
Comment (JosiasAurel) [inline README.md]: imo a simple "Build and run the docker image with ./build-with-docker.sh" will do

File: README.md [hunk 4]
```diff
@@ -8,6 +8,25 @@ However, on-device debugging is hard, so the engine can also be compiled to run
...
+
+
+### Native
```
Comment (JosiasAurel) [inline README.md]: i think this should be "Manual" and not "Native"

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

**Comment by whatwareweb** [comment]
OK done.

File: README.md (hunk 3 above)
Comment (JosiasAurel) [reply README.md]: also you will probably change into spade directory before changing code here

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

**Review by JosiasAurel — APPROVED** [review]

**Pull request merged**